src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,61 @@
-export const Pagination = () => {};
+import { PaginationItem } from '../PaginationItem';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  setCurrentPage: (page: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  setCurrentPage,
+}) => {
+  const itemsPerPage = Math.ceil(total / perPage);
```

Comment:
The variable name `itemsPerPage` is misleading because it suggests it holds the number of items per page, but it actually holds the total number of pages. Consider renaming it to `totalPages` to better reflect its purpose.